package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "aedes-tests",
+  "version": "0.0.1",
+  "description": "Integration/Black Box tests for Aedes MQTT Broker",
+  "main": "index.js",
+  "scripts": {
+    "unit": "tap --no-esm -J test/*.js",
```

Comment:
can you use a .taprc instead? I recently found out how useful they are.
Also disable flow, jsx and typescript.

Reply:
> Also disable flow, jsx and typescript.

Could you tell me how I can do this in package.json please?